toolkit/activity.py

```diff
@@ -302,7 +302,7 @@ def _sharing_setup(self):
         if self.shared_activity is None:
             logging.error('Failed to share or join activity')
             return
-        self._conn = self._shared_activity.telepathy_conn
+        self._conn = self.get_shared_activity.telepathy_conn
```

Comment:
I know you didn't make this change but this should be `self.shared_activity.telepathy_conn`.

Reply:
Done in f20f5ba

Reply:
Yeah, just saw it. Thanks.